docker/development/Dockerfile

```diff
@@ -45,20 +47,12 @@ RUN which yarn; yarn --version
 # yarn avaialble instead of our specified version
 RUN yarn set version "${YARN_VERSION}"
 
-# Setup Postgres repository
-# see https://wiki.postgresql.org/wiki/Apt
-RUN apt-get update
-RUN apt-get install -y lsb-release
-RUN curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor | tee /etc/apt/trusted.gpg.d/apt.postgresql.org.gpg >/dev/null
-RUN sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
-# update again with the new repository for postgres added
-RUN apt-get update
-RUN apt-get install -y postgresql-client-13
-
 # Install other dependencies
+# Note that postgresql-client-13 is available through debian bullseye
+# that the ruby image is based on
 RUN apt-get install -y --no-install-recommends \
```

Comment:
```suggestion
RUN apt update && \
    apt-get install -y --no-install-recommends \
```
You deleted all the `apt updates` above, one is needed tho:

```
 => ERROR [webpacker stage-1 11/18] RUN apt-get install -y --no-install-recommends         ffmpeg imagemagick pdftk ghostscript shared-mime-info         libarchive-tools postgresql-client-13 sqlite3 wget  1.0s
 => [mampf internal] load build context                                                                                                                                                                      0.3s
 => => transferring context: 385.32kB                                                                                                                                                                        0.0s
 => CACHED [webpacker build-pdfcomprezzor 2/5] WORKDIR /go/src                                                                                                                                               0.0s
 => CACHED [webpacker build-pdfcomprezzor 3/5] COPY pdfcomprezzor/go.mod pdfcomprezzor/go.sum pdfcomprezzor/main.go ./                                                                                       0.0s
 => CACHED [webpacker build-pdfcomprezzor 4/5] RUN GOOS=js GOARCH=wasm go build -o pdfcomprezzor.wasm                                                                                                        0.0s
 => CACHED [webpacker build-pdfcomprezzor 5/5] RUN cp "$(go env GOROOT)/misc/wasm/wasm_exec.js" .                                                                                                            0.0s
------
 > [webpacker stage-1 11/18] RUN apt-get install -y --no-install-recommends         ffmpeg imagemagick pdftk ghostscript shared-mime-info         libarchive-tools postgresql-client-13 sqlite3 wget wait-for-it:
0.812 Reading package lists...
0.823 Building dependency tree...
0.824 Reading state information...
0.825 Package ghostscript is not available, but is referred to by another package.
0.825 This may mean that the package is missing, has been obsoleted, or
0.825 is only available from another source
0.825
0.825 Package ffmpeg is not available, but is referred to by another package.
0.825 This may mean that the package is missing, has been obsoleted, or
0.825 is only available from another source
0.825
0.826 E: Package 'ffmpeg' has no installation candidate
0.826 E: Unable to locate package pdftk
0.826 E: Package 'ghostscript' has no installation candidate
0.826 E: Unable to locate package libarchive-tools
0.826 E: Unable to locate package postgresql-client-13
0.826 E: Unable to locate package sqlite3
0.826 E: Unable to locate package wait-for-it
------
failed to solve: process "/bin/bash --login -c apt-get install -y --no-install-recommends         ffmpeg imagemagick pdftk ghostscript shared-mime-info         libarchive-tools postgresql-client-13 sqlite3 wget wait-for-it" did not complete successfully: exit code: 100
```